src/test/java/teammates/test/cases/storage/FeedbackQuestionsDbTest.java

```diff
@@ -306,81 +314,89 @@ private FeedbackQuestionAttributes getNewFeedbackQuestionAttributes() {
         fqa.recipientType = FeedbackParticipantType.SELF;
         fqa.numberOfEntitiesToGiveFeedbackTo = 1;
         fqa.questionNumber = 1;
-        
+
         FeedbackTextQuestionDetails questionDetails = new FeedbackTextQuestionDetails("Question text.");
         fqa.questionType = FeedbackQuestionType.TEXT;
         fqa.setQuestionDetails(questionDetails);
-        
+
         fqa.showGiverNameTo =  new ArrayList<FeedbackParticipantType>();
         fqa.showRecipientNameTo = new ArrayList<FeedbackParticipantType>();
         fqa.showResponsesTo = new ArrayList<FeedbackParticipantType>();
-        
+
         return fqa;
     }
-    
+
     private List<FeedbackQuestionAttributes> createFeedbackQuestions(int num) throws Exception {
         FeedbackQuestionAttributes fqa;
         List<FeedbackQuestionAttributes> returnVal = new ArrayList<FeedbackQuestionAttributes>();
-        
-        for (int i = 1; i<=num ; i++) {
+
+        for (int i = 1; i <= num; i++) {
             fqa = getNewFeedbackQuestionAttributes();
             fqa.questionNumber = i;
             fqDb.createEntity(fqa);
             returnVal.add(fqa);
         }
-        
+
         return returnVal;
     }
-    
+
     private int[] createNewQuestionsForDifferentRecipientTypes() throws Exception {
-        
-        int[] num = new int[] {2 ,3 ,1 ,2};
+
+        int[] num = new int[] {
+                2,
+                3,
+                1,
+                2
+        };
```

Comment:
Not sure, I'll find out tomorrow and add a comment or change this part, magical indeed